Makefile

```diff
@@ -46,7 +46,7 @@ build: clean tidy kurator cluster-operator
 
 .PHONY: tidy
 tidy:
-	go mod tidy -compat=1.17
+	go mod tidy -go=1.19
```

Comment:
why do we need it?

Reply:
update upgrade capa to v2.0.0, we can remove this.